packages/extension/src/lib/extensionScripts.ts

```diff
@@ -1,16 +1,23 @@
 import browser from 'webextension-polyfill';
-import {
-  CreateRequestContentScripts,
-  contentScriptKey,
-} from '@dailydotdev/shared/src/hooks';
+import { contentScriptKey } from '@dailydotdev/shared/src/hooks';
 import { companionPermissionGrantedLink } from '@dailydotdev/shared/src/lib/constants';
-import { AnalyticsEvent } from '@dailydotdev/shared/src/lib/analytics';
+import { AnalyticsEvent as AnalyticsEventType } from '@dailydotdev/shared/src/lib/analytics';
```

Comment:
```suggestion
import { AnalyticsEvent as AnalyticsEventName } from '@dailydotdev/shared/src/lib/analytics';
```

Probably should have named these as such a long time ago 😮‍💨